DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -29,7 +31,7 @@ public string RegisterDelegate(DelegateRegistrationModel delegateRegistrationMod
                 CentreID = delegateRegistrationModel.Centre,
                 JobGroupID = delegateRegistrationModel.JobGroup,
                 Active = 1,
-                Approved = 0,
+                Approved = approved ? 1 : 0,

```

Comment:
Since this is always false at the moment I assume this is something that will actually do something in the future?

Reply:
Yeah, HEEDLS-402 covers adding cases where a new user is automatically approved.